public/app/features/explore/utils/ResultProcessor.ts

```diff
@@ -69,6 +69,8 @@ export class ResultProcessor {
       return null;
     }
 
+    // We are going from dataframe to table model and back because of the merging which is done on tableModel at the
+    // moment.
```

Comment:
We actually merged a DataFrame merge transform yesterday! so now we can look at getting rid of this costly DataFrame -> Table -> DataFrame mess :)